tests/unit/test_ps_model.py

```diff
@@ -130,6 +130,7 @@ def test_ps_model() -> None:
         mach_number,
         atyp_param.ff_idle_sls,
         atyp_param.ff_max_sls,
+        FlightPhase.CRUISE,
```

Comment:
@zebengberg Should this be just `FlightPhase` instead of `FlightPhase.CRUISE`? Because ps.correct_fuel_flow looks at FlightPhase.descent?

Reply:
We're okay here. You added a new parameter to `correct_fuel_flow`, so we need to put something here when we call the function in our tests. This parameter needs to be an instance of `FlightPhase` or an array of `FlightPhase` instances, not the type. Setting it to anything value besides `FlightPhase.descent` keeps the previous behavior, which is what we want to keep this test checking what it was designed for.